doc/nrf/links.txt

```diff
@@ -91,7 +91,6 @@
 .. _`SMP over Bluetooth`: https://github.com/apache/mynewt-mcumgr/blob/master/transport/smp-bluetooth.md
 
 .. _`Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/ee7ed2a/src/android/CHIPTool
-.. _`Commissioning nRF Connect Accessory using Android CHIPTool`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/docs/guides/nrfconnect_android_commissioning.md
 .. _`Working with Python Controller`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/docs/guides/python_chip_controller_building.md
 .. _`Performing Device Firmware Upgrade in Matter device`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/docs/guides/nrfconnect_examples_software_update.md
 .. _`Matter Protocol Overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/ee7ed2a/README.md
```

Comment:
ditto

Reply:
@LuDuda , this links to the main page of the Matter repository, but the description there is heavily outdated, to quote some Matter members involved in re-writing this page. Can we remove this link from ug_matter_architectures.rst?

Reply:
@greg-fer I'm fine with removing the sentence about this page in Matter architecture. Thanks.